ultralytics/engine/exporter.py

```diff
@@ -831,7 +837,8 @@ def export_saved_model(self, prefix=colorstr("TensorFlow SavedModel:")):
                 "tflite_support<=0.4.3" if IS_JETSON else "tflite_support",  # fix ImportError 'GLIBCXX_3.4.29'
                 "flatbuffers>=23.5.26,<100",  # update old 'flatbuffers' included inside tensorflow package
                 "onnxruntime-gpu" if cuda else "onnxruntime",
-            ),
+            )
+            + (("cmake", "onnxsim>=0.4.33") if self.end2end else ()),
```

Comment:
Unfortunately cmake must already be installed prior to onnxsim installation/compilation. We ditched onnxsim here on purpose to avoid these hassles, I'd suggest avoiding it to provide uniform results across operating systems and installations.